TestFiles/app-components-test.yaml

```diff
@@ -0,0 +1,25 @@
+version: v0.1
+testName: basicAppComponentsTest-edit
+displayName: E2E test for app components
+testPlan: ./../ConfigFiles/BasicTest.jmx
+testType: JMX
+description: 'E2E for the build pipeline (JMX)'
+autoStop:
+  errorPercentage: 10
+  timeWindow: 20
+appComponents:
+- resourceId: "/subscriptions/7c71b563-0dc0-4bc0-bcf6-06f8f0516c7a/resourceGroups/adonew-sampleapp-rg/providers/Microsoft.Web/serverfarms/adonew-sampleapp-host"
```

Comment:
Should we use integration test resources here?

Reply:
These resources might get deleted in future

Reply:
done.